site/source/tutorials/1010_containers_and_injecting_state/demo/finished/biz-e-corp/src/ApplicationContext.ts

```diff
@@ -1,10 +1,10 @@
+import { Evented } from '@dojo/widget-core/Evented';
```

Comment:
This isn’t from widget core is it?

Reply:
Ugh. Nope. The best mistakes are the most obvious. Fixed.